src/main/java/uk/gov/pay/api/filter/ratelimit/RateLimiter.java

```diff
@@ -17,13 +18,13 @@ public RateLimiter(LocalRateLimiter localRateLimiter, RedisRateLimiter redisRate
         this.redisRateLimiter = redisRateLimiter;
     }
 
-    public void checkRateOf(String accountId, String key, String method) throws RateLimitException {
+    public void checkRateOf(String accountId, RateLimiterKey key, String method) throws RateLimitException {
         try {
-            redisRateLimiter.checkRateOf(accountId, key, method);
+            redisRateLimiter.checkRateOf(accountId, key);
         } catch (RedisException e) {
             LOGGER.warn("Exception occurred checking rate limits using RedisRateLimiter, falling back to LocalRateLimiter");
 
-            localRateLimiter.checkRateOf(accountId, key, method);
+            localRateLimiter.checkRateOf(accountId, key.getKey(), method);
```

Comment:
`localRateLimiter.checkRateOf(accountId, key.getKey(), key.getMethod());`?
Then you don't need to pass in method into this method (if you se what I mean)

Reply:
Oh yeah, fair enough. I've added `method` later and forgot about it in the clean-up phase.

Reply:
Updated